src/utils/isAllowed.js

```diff
@@ -9,10 +9,20 @@ const isAllowed = async (to) => {
     message = `Please enter valid Ethereum Wallet Address`
     return { message, status }
   }
-  const oceanBalance = await getOceanBalance(to)
-  const EthBalance = await getEthBalance(to)
-  const EthBalanceLimit = web3.utils.toWei(process.env.TOKEN_AMOUNT, 'ether')
-  const OceanBalanceLimit = web3.utils.toWei(process.env.TOKEN_AMOUNT, 'ether')
+  const oceanBalance = BigInt(await getOceanBalance(to))
```

Comment:
The oceanBalance and oceanBalanceLimit were retrived as strings. When comparing, we were comparing the strings values instead of integer values.